yarn.lock

```diff
@@ -12463,6 +12463,13 @@ tslint-loader@3.6.0:
     rimraf "^2.4.4"
     semver "^5.3.0"
 
+tslint-no-unused-expression-chai@0.1.4:
```

Comment:
Are the changes in this file intended?

Reply:
I think the `develop` branch did not have the latest `yarn.lock`, hence the differences that got generated for me locally. I just merged `develop` and now there are no more `yarn.lock` changes.

Reply:
Alright, thanks for looking into it!